android/app/src/main/java/org/fossasia/openevent/api/DataDownloadManager.java

```diff
@@ -29,28 +40,38 @@ public static DataDownloadManager getInstance() {
     }
 
     public void downloadEvents() {
-        client.getOpenEventAPI().getEvents().enqueue(new EventListResponseProcessor());
+        openEventClient.getOpenEventAPI().getEvents().enqueue(new EventListResponseProcessor());
     }
 
     public void downloadSpeakers() {
-        client.getOpenEventAPI().getSpeakers().enqueue(new SpeakerListResponseProcessor());
+        openEventClient.getOpenEventAPI().getSpeakers().enqueue(new SpeakerListResponseProcessor());
     }
 
     public void downloadSponsors() {
-        client.getOpenEventAPI().getSponsors().enqueue(new SponsorListResponseProcessor());
+        openEventClient.getOpenEventAPI().getSponsors().enqueue(new SponsorListResponseProcessor());
     }
 
     public void downloadSession() {
-        client.getOpenEventAPI().getSessions("start_time.asc").enqueue(new SessionListResponseProcessor());
+        openEventClient.getOpenEventAPI().getSessions("start_time.asc").enqueue(new SessionListResponseProcessor());
     }
 
     public void downloadTracks() {
-        client.getOpenEventAPI().getTracks().enqueue(new TrackListResponseProcessor());
+        openEventClient.getOpenEventAPI().getTracks().enqueue(new TrackListResponseProcessor());
     }
 
     public void downloadMicrolocations() {
-        client.getOpenEventAPI().getMicrolocations().enqueue(new MicrolocationListResponseProcessor());
+        openEventClient.getOpenEventAPI().getMicrolocations().enqueue(new MicrolocationListResponseProcessor());
     }
 
+    public void downloadPageId(Context context, String eventName, String accessToken) {
+        graphClient.getFacebookGraphAPI().getPageId(eventName, accessToken).enqueue(new PageIdResponseProcessor(context));
+    }
 
+    public Observable<Feed> downloadFeed(String pageId, String fields, String accessToken) {
```

Comment:
This shouldn't be here. You are using event bus and still returning feed? Why?

This should be present wherever you need to get the posts.

Same with the function above